js/languages/en-US.json

```diff
@@ -11,17 +11,20 @@
   "testnet": "Test Mode",
   "testnetTooltip": "Test Mode uses testnet coins, which are intended for testing purposes only.",
   "tabMenuHeading": "MENU",
+  "fiatBtcPairing": "%{fiatAmount} (%{btcAmount})",
```

Comment:
I moved these three out to re-use globally. I can't imagine these being context specific and it seemed unnecessarily repetitive to keep re-declaring them.